lib/agent/main.go

```diff
@@ -9,13 +9,35 @@ import (
 	"main/machine"
 )
 import (
-	"main/cloud"
-	"main/rate_limiting"
+	. "main/aikido_types"
+	"main/server_utils"
+	"main/utils"
 	"os"
 	"os/signal"
+	"sync/atomic"
 	"syscall"
+	"time"
 )
 
+var serversCleanupChannel = make(chan struct{})
+var serversCleanupTicker = time.NewTicker(10 * time.Minute)
+
+func serversCleanupRoutine(server *ServerData) {
+	for _, token := range globals.GetServersTokens() {
+		server := globals.GetServer(token)
+		if server == nil {
+			continue
+		}
+		now := utils.GetTime()
+		lastConnectionTime := atomic.LoadInt64(&server.LastConnectionTime)
+		if now-lastConnectionTime > MinServerInactivityForCleanup {
+			// Server has been inactive for more than 10 minutes
+			log.Infof("Server has been inactive for more than 10 minutes, unregistering...")
+			server_utils.Unregister(token)
```

Comment:
There's a brief chance that we might delete a just created server because `LastConnectionTime` is `0`  until `GetCloudConfig` is called? Should we set `LastConnectionTime` on creation of a server?